packages/rrweb/src/record/index.ts

```diff
@@ -27,6 +27,7 @@ import {
 import { IframeManager } from './iframe-manager';
 import { ShadowDomManager } from './shadow-dom-manager';
 import { CanvasManager } from './observers/canvas/canvas-manager';
+import { callbackWrapper } from '../sentry/callbackWrapper';
```

Comment:
I think this should be generally upstreamable, so I would try to name this in a way that is not sentry specific.

Reply:
Actually disregard this. I can see we need to separately upstream this with the amount of pending upstream changes already.

Reply:
Yeah, I'm actually thinking to upstream this as a more generic configurable `errorHandler` kind of thing. IMHO that probably makes the most sense from a generic rrweb usage perspective! Then we can configure the error handler to add the `__rrweb__` stuff in sentry-javascript for us (long term, when we eventually update our fork to v2)